Cargo.toml

```diff
@@ -21,7 +21,7 @@ license = "ISC"
 name = "rustls-webpki"
 readme = "README.md"
 repository = "https://github.com/rustls/webpki"
-version = "0.102.1"
+version = "0.102.2"
```

Comment:
Note: the version in the commit message is wrong.

Reply:
Good catch!